examples/gno.land/r/gnoland/faucet/admin.gno

```diff
@@ -5,44 +5,77 @@ import (
 	"std"
 )
 
-func AdminSetInPause(inPause bool) error {
+func AdminSetInPause(inPause bool) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
 	gInPause = inPause
-	return nil
+	return ""
 }
 
-func AdminSetMessage(message string) error {
+func AdminSetMessage(message string) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
 	gMessage = message
-	return nil
+	return ""
 }
 
-func AdminSetPerTransferSend(send std.Coins) error {
+func AdminSetTransferLimit(amount int64) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
-	gPerTransferSend = send
-	return nil
+	gLimit = std.Coin{Denom: "ugnot", Amount: amount}
+	return ""
 }
 
-func AdminSetAdminAddr(addr std.Address) error {
+func AdminSetAdminAddr(addr std.Address) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
 	gAdminAddr = addr
-	return nil
+	return ""
 }
 
-func AdminSetControllerAddr(addr std.Address) error {
+func AdminAddController(addr std.Address) string {
 	if err := assertIsAdmin(); err != nil {
-		return err
+		return err.Error()
 	}
-	gControllerAddr = addr
-	return nil
+
+	size := gControllers.Size()
+
+	if size >= gControllersSize {
+		return "can not add more controllers than allowed"
+	}
```

Comment:
You can inline this check:

```go
	if gControllers.Size() >= gControllersSize {
		return "can not add more controllers than allowed"
	}
```

Reply:
What is the benefit of writing it in line? It is a lot easier to read with a shorter comparison statement.